cli.php

```diff
@@ -11,6 +11,10 @@ class WPSuperCache_Command extends WP_CLI_Command {
 	 * @synopsis [--post_id=<post-id>] [--permalink=<permalink>]
 	 */
 	function flush( $args = array(), $assoc_args = array() ) {
+		require_once( WPCACHEHOME . '/wp-cache-phase1.php' );
+		global $WPSC_HTTP_HOST;
+		$WPSC_HTTP_HOST = parse_url(get_site_url())['host'];
```

Comment:
If you set `url` in your wp-cli.yml
```yaml
url: https://my-domain.com/
```
You'll get  set `HTTP_HOST` set
https://github.com/wp-cli/wp-cli/blob/6e626cc73e0463fe9564df3b98c9f08f9466b430/php/class-wp-cli.php#L105-L1162

(Using backticks `\`` make your comments more readable.)